src/app/ReadClient.h

```diff
@@ -126,6 +127,16 @@ class ReadClient : public Messaging::ExchangeDelegate
          */
         virtual void OnSubscriptionEstablished(SubscriptionId aSubscriptionId) {}
 
+        /**
+         * OnResubscriptionAttempt will be called when a re-subscription has been scheduled.
```

Comment:
"OnResubscriptionAttempt will be called when a re-subscription has been scheduled as a result of the termination of an in-progress or previously active subscription."

Reply:
i.e It's good to to be explicitly clear about the conditions that arise in this call being made.